.gitignore

```diff
@@ -9,3 +9,9 @@ libre.a
 libre.dylib
 libre.pc
 libre.so
+
+# ctags
+tags
+
+# Vim swp files
+*.swp
```

Comment:
Please include only related changes within a pull a request and make a separat for these. Thanks.

Reply:
done.